### ISSUE TITLE
Doesn't appear in KDE media center.

### ISSUE THREAD

**Issue by Bitwolfies** [jellyfin/jellyfin-media-player#185]
On KDE, Jellyfin Media player will not appear in the meida player applet.

**To Reproduce**
1. On KDE, start the app, play something
2. See it does not appear in the media applet.
**Expected behavior**
It to appear in the media applet.

**Desktop (please complete the following information):**
 - OS: EndeavorOS 
 - Version: JF media player 1.6.1-1
 - Installation Method: AUR

**Comment by iwalton3** [comment]
Likely a duplicate of https://github.com/jellyfin/jellyfin-media-player/issues/3

**Comment by Bitwolfies** [comment]
Yeah, youre right, the fix in that thread fixed it 90% of the way, everything now works but thumbnails. Dunno how possible it is to include said fix into the next version however.
